### ISSUE TITLE
outdated npm won't let ember generate new project

### ISSUE THREAD

**Issue by gossi** [ember-cli/ember-cli#7510]
I just upgraded `ember-cli` to `2.17.1`. Here is my setup:

```
$ ember version --verbose && npm --version && yarn --version
ember-cli: 2.17.1
http_parser: 2.7.0
node: 8.9.3
v8: 6.1.534.48
uv: 1.15.0
zlib: 1.2.11
ares: 1.10.1-DEV
modules: 57
nghttp2: 1.25.0
openssl: 1.0.2n
icu: 60.1
unicode: 10.0
cldr: 32.0
tz: 2017c
os: darwin x64
2.15.12
1.3.2
```

Well, at the moment, I haven't updated node/npm (as they are bundled with homebrew for me), since corber blocks node@9 for me and I'm waiting on this one. Though when running `ember new project-name` I do get this message:

```
Error creating new application. Removing generated directory `./project-name`
Ember CLI is now using the global npm, but your npm version is outdated.
Please update your global npm version by running: npm install -g npm
```

Yes, my system isn't upgraded to the newsest npm version yet but I don't see a reason for this to block the `ember new` command and this error message. Is this an overly careful error message?

**Comment by ro0gr** [comment]
As a workaround you can run "new" command with  "--skip-mom" flag. After ember-cli generated file structure you have to install dependencies manually. 

If the error occurs on npm install step we can at least get an actual error message.

You can also use --yarn flag.

**Comment by gossi** [comment]
Got help in slack. It was a very weird symlink to an old npm version. Problem is solved.

**Comment by typeoneerror** [comment]
@gossi any chance you can share the solution? also running into this.

**Comment by gossi** [comment]
Uuuh, IIRC my npm wasn't build properly. So I had an old symlink `/usr/local/bin/npm` which led to a non-existent target or something. It turns out I had a broken npm installation. I ended up cleaning my system install, did a fresh (re)install of all that and got it working again. Meanwhile, I've upgraded to the current version of node and npm, since all my dependencies did this step as well and everything is running smoothly for me. Hope it helps.
